2/contrib/openshift/base-plugins.txt

```diff
@@ -1,4 +1,4 @@
-openshift-login:1.0.17
+openshift-login:1.0.19
```

Comment:
fyi in v1.0.18 we got an upstream contribution for proxy based prefix overrides (a jenkins thing) to play nice with our oauth integration